conans/client/command.py

```diff
@@ -735,7 +735,8 @@ def run(self, *args):
             self._user_io.out.error(msg)
         except Exception as exc:
             import traceback
-            print(traceback.format_exc())
+            # print(traceback.format_exc())
```

Comment:
But it is uncommented right now in develop, so I was wondering why you commented it in this PR. Yes, because of the encoding issues it could be problematic, maybe leave it anyway, that it might help debugging other bugs, even if it will eventually crashed (it had already crashed, as we are in Exception-not controlled except)